docs/RegisteredFilterPlugins.md

```diff
@@ -13,7 +13,10 @@ To register a filter please contact The HDF Helpdesk with the following informat
 * Links to any relevant information including licensing information
 
 Here is the current policy regarding filter identifier assignment:
-The filter identifier is designed to be a unique identifier for the filter. Values from `zero` through `32,767` are reserved for filters supported by The HDF Group in the HDF5 library and for filters requested and supported by the 3rd party.
+The filter identifier is designed to be a unique identifier for the filter.
+Values from `0` to `255` are reserved for internal use by The HDF Group for filters that are part of the HDF5 library itself.
+Values from `256` to `511` are available for testing new VOL connectors.
```

Comment:
This document is about filter plugins, not VOL connectors.

Reply:
Ah, yes, I apologize for my mistake; I have had the VOLs on my mind lately. Here is what we state in the Doxygen documentation if you would like to use the same wording:

Values from zero through 32,767 are reserved for filters supported by The HDF Group in the HDF5 library and for filters requested and supported by 3rd parties. Values from 32768 to 65535 are reserved for non-distributed uses (e.g., internal company usage) or application usage when testing a feature.